modules/installation-rhv-building-ignition-files.adoc

```diff
@@ -5,12 +5,12 @@
 [id="installation-rhv-building-ignition-files_{context}"]
 = Building the ignition files
 
-To build the Ignition files from the manifest files you just generated and modified, you run the installation program. This action creates a {op-system-first} machine, `initramfs`, which fetches the ignition files and performs the configurations needed to bring a node to life.
+To build the Ignition files from the manifest files you just generated and modified, you run the installation program. This action creates a {op-system-first} machine, `initramfs`, which fetches the Ignition files and performs the configurations needed to create a node.
```

Comment:
Bringing a node to life sounds awesome; maybe we should keep it, just this once?